### DISCUSSION TITLE
Add option to disable nested entries in dropdown

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15826] [Ideas]
It would be great if you could disable/enable a nested entry directly from this action menu. Right now, you have top open the entry in order to do so.

![CleanShot 2024-10-02 at 10 20 15@2x](https://github.com/user-attachments/assets/b0a405c9-df09-4615-9a8a-4fe641416b97)

This is currently an option in Neo which is what we're migrating away from.

![CleanShot 2024-10-02 at 10 23 48@2x](https://github.com/user-attachments/assets/aab51b0f-d4e7-4037-905e-54e7d28b02be)

**Comment by Nellyaa** [comment]
Yeah, this would be great! Also copying and pasting elements from this context menu would be marvelous... :)

**Comment by ryanleichty** [comment]
I added this last year but wanting to highlight it again as something that's still relevant today.
